Add '--enable-plugins' option to 'start' CLI command (#1409)
kuzzleio/kuzzle: doc/1/guides/essentials/cli/index.md

}
```

#### `--enable--plugins`
<SinceBadge version="1.10.0" />

> Enable given plugins. If multiple names are given, they need to be separated with comma:

Reviewer: ```suggestion
Enable given plugins. If several names are given, they need to be separated with a comma:
```

}
```

#### `--enable--plugins`

> <SinceBadge version="1.10.0" />

Reviewer 2: Hmm... did we add new features in 1-dev recently? The next v1 version may be 1.9.3.

```suggestion
<SinceBadge version="1.10.0" />
```

```bash
./bin/kuzzle start --enable-plugins kuzzle-custom-plugin-one,kuzzle-custom-plugin-two
```

> Enable given plugins. If several names are given, they need to be separated with comma.

Reviewer 2: Is this using the directory name, or the manifest.json one? 

```suggestion
Enable given plugins. If several names are given, they need to be separated with commas.
```
